### PR TITLE
Address page - staking table

### PR DESCRIPTION
**Pull request by potvik**

### CODE REVIEW

File: src/components/AddressPage.vue [hunk 1]
```diff
@@ -111,21 +120,39 @@ export default {
...
               txs[tx.hash] = tx;
             });
           });
 
+          address.shardData.forEach(data => {
+            data.stakingTxs.forEach(tx => {
+              stakingTxs[tx.hash] = {
+                ...tx,
+                delegator: tx.msg.delegatorAddress,
+                validator: tx.msg.validatorAddress,
```
Comment (denniswon) [inline src/components/AddressPage.vue:141]: we also need to show the type of txn for staking txn.

File: src/explorer/service.js [hunk 2]
```diff
@@ -10,8 +10,8 @@ Promise.prototype.delay = function(time) {
   });
 };
 
-// const BACKEND_URL = `${'explorer.testnet.harmony.one'}:8888`;
-const BACKEND_URL = `${window.location.hostname}:8888`;
+const BACKEND_URL = `${'explorer.os.hmny.io'}:8888`;
+// const BACKEND_URL = `${window.location.hostname}:8888`;
```
Comment (denniswon) [inline src/explorer/service.js:14]: revert this back

File: src/filter.js [hunk 3]
```diff
@@ -34,6 +34,10 @@ export function formatAge(timestamp) {
 }
 
 export function formatAmount(amount) {
+  if (amount === undefined) {
+    return '-';
+  }
+
```
Comment (denniswon) [inline src/filter.js:40]: don't return string here? hmm this is when I get to not like js (compared to ts). shrug...

Comment (potvik) [reply src/filter.js:40]: Intl.NumberFormat('en-US', { maximumFractionDigits: 18 }).format(fee)

also return string

**Pull request merged**